ACCESSIBILITY.md

```diff
@@ -0,0 +1,97 @@
+
+1. **Removing barriers**: We actively advocate for and employ tactics that seek to remove barriers for any current or future participants in our community. This may include people who historically have been limited or structurely excluded from participating in through educational, geographical, linguistic, or technical barriers.
+2. **Sharing openly and with openness**: We aim to draft, write, review, and edit our open access guides using open source practices, and we do this in order to enable people from a broad range of backgrounds to get involved in the project.
+3. **Supporting inclusive participation**: We acknowledge that openness alone does not make a project accessible in a broad sense, and we are committed to maintaing a socio-technical infrastructure that enables inclusive participation more broadly. This includes hosting spaces for contribution, documenting our practices, and actively gathering feedback.
+
+The practices and policies listed here relate to these three operating principles, which shape how we work within the community to enable access on the web, on our community platforms, and while planning events.
+
+### 2.1 Web Accessibility
+
+_The Turing Way_ is committed to ensuring the web accessibility of our guides.
+
+As an open source community that relies on  projects and platforms for our underlying infrastructure, we acknowledge that we are limited in what standalone access-centered support we can offer. Our project is built with JupyterBook and using the Github platform. Both of these platforms have policies that address how they address the [W3C WAI's Web Content Accessibility Guidelines 2.0](https://www.w3.org/TR/WCAG/). You can read more about the the experience of a researcher at Accessible Notebooks Hackathon in ["Notebooks For All: Making Jupyter Notebooks More Accessible"](https://astrobites.org/2023/06/10/notebooks-for-all/) and about Github's work in the [Github Accessibility Report](https://accessibility.github.com/).
+
+As a community however, we are committed to ensuring broader accessibility of our resources through using this infrastructure through:
+- Choosing sufficient color contrast in our hand-drawn images that make our images accessible in the context of color-blindness
+- Using and employing appropriate and relevant alt text (also known as 'alternative text') for all of our images, charts, and data visualisations (an ongoing project with in the community)
+- Navigation without the need for a mouse or track pad during either the reading process on JupyterBook and/or contributing process on Github
+
+### 2.2 Community Platform Accessibility
+
+_The Turing Way_ is committed to ensuring our accessibility as a community on all the platforms we use.
+
+In our Community Handbook, we describe these practices more specifically and in more detail. But as a baseline, our community is dedicated to enabling access across the various platforms we use through:
```

Comment:
Can we add a link through to the accessibility comms platform page? As this seems to be a summary of that content.

Reply:
Yes - added a link and description here to the Community Handbook.